cmd/buildah/common_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"os/user"
+	"testing"
+
+	"flag"
+
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/urfave/cli"
+)
+
+func TestGetStore(t *testing.T) {
+	u, err := user.Current()
+	if err != nil {
+		t.Log("Could not determine user.  Running as root may cause tests to fail")
```

Comment:
Here you're saying you can't run as root, but at line 19 you're saying you must run at root in the else for this if?